lib/generators/pageflow/install/install_generator.rb

```diff
@@ -8,6 +8,7 @@ def generate_all
         generate 'active_admin:install User --skip-users'
         generate 'active_admin:devise User --no-default-user'
         generate 'friendly_id'
+        generate 'kaminari:config'
```

Comment:
What exactly can be configured in the generated file? I'm a bit worried, that changes made in a host application to change Active Admin behavior might unintentionally also change the feed output. Can we somehow decouple from these defaults by making explicit choices in Pageflow (and provide Pageflow config options where necessary)? I'd prefer if Kaminari did not become part of our API surface.

Reply:
I believe it's just view helpers and defaults, and we can safely omit this step.

However with this change we _are_ pulling Kaminari on board, and the kaminari generator is something people will be able to do if they want. Luckily we don't use paging in any html view and we can sort of pretend that Kaminari isn't really here.